libbfio-sys/build.rs

```diff
@@ -66,55 +69,189 @@ fn build_static() -> Option<PathBuf> {
         target.join("lib").display()
     );
 
-    println!("cargo:rustc-link-lib=static=bfio");
     println!(
         "cargo:rustc-link-search=native={}",
         target.join("lib").canonicalize().unwrap().to_string_lossy()
     );
 
-    Some(target)
+    target.join("include")
 }
 
-fn link_dynamic() -> Option<PathBuf> {
-    println!("cargo:rustc-link-lib=dylib=bfio");
+/// Build the lib on windows (using msbuild and libyal's vstools).
+/// Require python to be installed.
+/// This function will also add the needed folder to the `link-search` path.
+/// Return the "include" folder for the library (to be used by bindgen).
+#[cfg(target_os = "windows")]
+fn build_lib(lib_path: PathBuf, shared: bool) -> PathBuf {
+    let python_exec =
+        env::var("PYTHON_SYS_EXECUTABLE ").unwrap_or_else(|_| "python.exe".to_owned());
```

Comment:
also - does this work with python3?

we should probably complain with a nice message if python is not found.